core/utils/poly-look/src/visualisations/charts/bubble-charts/bubbleCluster/bubbleCluster.js

```diff
@@ -28,12 +30,14 @@ const defaultOnClickFunction = () => {};
  * @param {string} data[].title - The title/name the bubble has
  * @param {number} data[].value - The value of the bubble, which corresponds to it's radius
  * @param {number} data[].icon - A svg that should be displayed as the bubble in string form as you would get it from rollup-svg-plugin (eg. "<svg>...</svg>")
+ * @param {Object[]} data[].children - Data for nested bubbles - only one level of nesting is currently supported, i.e. no grandchildren.
  * @param {number = 400} width - The width of the svg
  * @param {number = 300} height - The height of the svg
  * @param {string|callback = "blue"} [bubbleColor] - The color of the bubble (callbacks receive event and data)
+ * @param {string|callback = "#f7fafc"} [strokeColor] - The color of the bubble outline (callbacks receive event and data)
  * @param {string|callback = "white"} [textColor] - The color of the bubble text (callbacks receive event and data)
  * @param {number|callback = 1} [opacity] - The opacity of the bubbles color 0 <= opacity <= 1 (callbacks receive event and data)
- * @param {boolean = true} [showValues] - Whether texts displaying the value of the bubble are added
+ * @param {string|callback} [text] - The text rendered on top of each bubble, by default its value (callbacks receive event and data)
```

Comment:
Is it just on top now?

Reply:
On top in terms of the Z axis, not Y :D

Reply:
This is correctly documented as a string or a callback, but is the code actually taking that into account?